misc/thesis-fields.tex

```diff
@@ -38,11 +38,11 @@
 חלק מן התוצאות בחיבור זה פורסמו כמאמרים מאת המחבר ושותפיו למחקר בכנסים ובכתבי-עת במהלך תקופת מחקר הדוקטורט של המחבר, אשר גרסאותיהם העדכניות ביותר הינן:%
 
 \begin{otherlanguage}{english}%
-% No need to mention the bibliography file this time, as it has already been used in
-% the English invocation
-\butcheredbibliography{pubinfo}{front/pubinfo}
+\butcheredbibliography{front/pubinfo.bib}
 \end{otherlanguage}%
 }
 
-\thesisbibfiles{back/general}
-\thesisbibstyle{alpha}
+% The biblatex package uses the \addbibresource to add bib files to its database. 
+\addbibresource{back/general.bib}
+% TODO: the style seems like a package option; not sure how to pass it from here.
+% \thesisbibstyle{trad-alpha}
```

Comment:
Not sure I'm following. Can we not set the style of different bibliographies within the same document? That would be bad...

Reply:
It's possible, I was able to do that for the pubinfo bibliography. I just need to pass it early enough I guess.

Reply:
How early? Do we need to add a comment about this somewhere?

Reply:
According to this [StackExchange question](https://tex.stackexchange.com/questions/299064/biblatex-two-bibliographies-with-different-styles-and-sortings), it could be possible to change styles without passing a package option. However, the answer seems to require copying the desired style into the sources somehow. 

The [user manual](https://ctan.mines-albi.fr/macros/latex/contrib/biblatex/doc/biblatex.pdf) says the only way to pass the `style`/`bibstyle`/`citestyle` options is at package load time (Appendix E).

I guess we could either pass the options as extra options to the iitthesis class, or defer the package load through an additional macro or `\AtEndPreamble`.